src/main/java/live/einfachgustaf/mods/smp/mixins/MixinResultSlot.java

```diff
@@ -5,16 +5,33 @@
 import net.minecraft.world.entity.player.Player;
 import net.minecraft.world.inventory.ResultSlot;
 import net.minecraft.world.item.ItemStack;
+import org.spongepowered.asm.mixin.Final;
 import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(ResultSlot.class)
 public class MixinResultSlot {
 
+    @Shadow
+    @Final
+    private Player player;
+
+    private boolean wasQuickCrafted = false;
+
     @Inject(at = @At("HEAD"), method = "onTake")
     public void injectSlotsChanged(Player player, ItemStack itemStack, CallbackInfo ci) {
```

Comment:
please rename the method to injectOnTake

Reply:
This is how I got it from main

Reply:
oh, than it's my fault. sorry